src/index.ts

```diff
@@ -123,16 +134,56 @@ export const Rune: RuneExport = {
     Rune.gameOver = function () {
       const score = Rune._getScore()
       Rune._validateScore(score)
-      globalThis.postRuneEvent?.({ type: "GAME_OVER", score, challengeNumber: Rune.getChallengeNumber() })
+
+      // Reset randomness to be deterministic across plays
+      const challengeNumber = Rune.getChallengeNumber()
+      Rune.deterministicRandom = Rune._randomNumberGenerator(challengeNumber)
+
+      globalThis.postRuneEvent?.({ type: "GAME_OVER", score, challengeNumber })
```

Comment:
A bit messy to have to duplicate this with actual `gameOver()`, but couldn't think of a cleaner way w/o adding complexity.

Reply:
Fixed by @mattbretl with https://github.com/rune/rune-games-sdk/pull/38/commits/46ad431e06aa54e01bda9529f5760279942f8862.